lib/rspec-puppet/support.rb

```diff
@@ -380,7 +393,7 @@ def with_vardir
       end
     end
 
-    def build_catalog_without_cache(nodename, facts_val, trusted_facts_val, hiera_config_val, code, exported, node_params, *_)
+    def build_catalog_without_cache(nodename, facts_val, trusted_facts_val, hiera_config_val, code, exported, node_params, *args)
```

Comment:
I can move `external_trusted_data` to come after `trusted_facts_val` instead. Just wasn't sure if doing so would break existing users of this gem and if that is an issue.

Reply:
I guess since `build_catalog` just passes through its `args` splat, changing the signature of this method would effectively change the signature of `build_catalog` as well...